### ISSUE TITLE
[$60][MSFT-199] All pages: Inconsistent landmarks across pages.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2864]
**Steps to Reproduce**
1. Go to the sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure that the screen reader is switched on. 3. Press the letter 'r' a couple of times to navigate to the various landmarks on the page or press 'ctrl + insert + r' to view the list of landmarks on the page. 4. Note that there is no 'banner' landmark for the header, 'main landmark' for the main content of the page, and 'contentinfo landmark' for the footer of the page. 5. Also note that this issue can be found on the other pages too. The landmarks are inconsistent.

**Expected Result**
The content on the page should be placed inside the relevant landmark roles for screen reader users to skip to the content in the respective landmark he/she intends to read. The banner landmark should contain the header of the page. The main landmark should be wrapped around the main content of the page. The contentinfo should be used to contain the footer of the page. Navigation, search etc can be used additionally.

**Actual Result**
There are no banner, main and contentinfo landmarks on the page.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.1 Bypass Blocks
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2034

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096162 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096162 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096162 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096162 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2864 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2864 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2864 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2864 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2864 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097365 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097365 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097365 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by Oanh-and-only-Oanh** [comment]
Per Tony's comment: "I think that landmark thing is probably true for the listing page too if it’s not already called out.  i.e. It isn’t obvious that the page is sectioned into filters, side nav, and listing"

**Comment by drasticdpk** [comment]
@Oanh-and-only-Oanh yes we are having this landmark on listing page too.
![Landmark](https://user-images.githubusercontent.com/25297859/62230455-8d3b5480-b3e1-11e9-9ceb-7d909ec7dc25.png)

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097365<br/><br/>```This is an automated message for crazyk via Topcoder X```
